spec/index.html

```diff
@@ -1927,16 +1927,18 @@ <h2>Replacing Blank Nodes with IRIs</h2>
     mint a new, globally
     unique IRI (a <dfn class="export">Skolem IRI</dfn>) for each blank node so replaced.</p>
 
-  <p>This transformation does not appreciably change the meaning of an
-    RDF graph, provided that the Skolem IRIs do not occur anywhere else.
-    It does however permit other graphs
+  <p>This transformation produces a new graph which entails the original one (see also <a data-cite="RDF12-SEMANTICS#skolemization">Skolemization</a>).
+    In this context it is important to note that Skolem IRIs and blank nodes are different in nature: while IRIs directly represent resources, blank nodes only ensure 
+    the existence of a resource. By producing Skolem IRIs we create concrete witnesses of this exitence.    
```

Comment:
```suggestion
    the existence of a resource. By producing Skolem IRIs we create concrete witnesses of this existence.    
```
(Also, I'm not accustomed to "witnesses" in the context of RDF; but as it is formally correct, if editors are happy I'll adapt.)

Reply:
mmm, good point. "Witness" is the term you normally use in context with skolem functions, I just adopted from there. But we can change if you want.
(thank you for fixing my spelling mistakes! :) )